docs/docs/platform/quickstarts/nextjs.mdx

```diff
@@ -0,0 +1,619 @@
+---
```

Comment:
I think we should move our quickstart guides to a separate page. Just like [here](https://testing-library.com/docs/recipes).

Reply:
I think we should discuss these types of bigger changes in a separate issue.